src/lib/core/media-marshaller/media-marshaller.ts

```diff
@@ -247,6 +247,21 @@ export class MediaMarshaller {
     }
   }
 
+  /**
+   * trigger an update for a given element and key (e.g. layout)
+   * @param element
+   * @param key
+   */
+  triggerUpdate(element: HTMLElement, key?: string): void {
```

Comment:
That seems maybe a little harsh? I was keeping this in sync with the `BaseDirective2::triggerUpdate` terminology, since it's basically the same idea. The long-term plan is to move that to use this method, but for some reason it wasn't working for this PR.

Reply:
These changes now look good: consistent usages.